packages/common/client-utils/api-report/client-utils.beta.api.md

```diff
@@ -1,13 +0,0 @@
-## Beta API Report File for "@fluid-internal/client-utils"
```

Comment:
When cleaning up the extra alpha reports I generated in an earlier version of this report, I deleted all the report files and regenerated them all, and turns out these reports were still checked in but not getting generated, so they are now removed. This seems like an improvement so I left it in the PR

Reply:
I was looking before I saw this comment. There are scripts that should be generating reports. Something is wrong and we'll need to follow up. Please restore them for now.

Reply:
I put back the likely out of date files which the build does not update.

Reply:
#22489
(someone forgot an 's')